### PR TITLE
submitting Week 6 assignment

### PR DESCRIPTION
**Pull request by HenrikeW**
Link to my deployed project: https://henrikes-weather.netlify.app/

My reflections on how this week's project turned out: I'm happy with the outcome and feel that I've learned a lot about functions in JS and how to write shorter code. I only touched on the red level with the icons but would have done more if I had had more time. It didn't seem too difficult. In the blue level, I chose to show the min/max temperature for every day instead of choosing one specific time of day as suggested in the hint. That took me a while and felt like I created my own red level there :) 

Things I'd like to have clarified or explained in more detail: nothing specific

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+fetch(API_URL_TODAY)
+    .then((response) => {
+        return response.json();
+    })
+    .then((json) => {
+        updateWeatherToday(json);
+    });
+
+    
+fetch(API_URL_FORECAST)
```
Comment (pwangy) [inline code/script.js:25]: :heart_eyes: Henrike! These fetchs are so clean! <3

**Review by pwangy — APPROVED** [review]
Henrike, I can't say I have very much feedback for you this week. Your code is super clean, easy to read, and consistent in your use of semicolons. I also like how the layout changes from vertical in mobile to horizontal on desktop. It's so neat to get to see how someone else has organized their code and your work this week has inspired me to try something the next time around. Well done!

File: code/index.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+</head>
+<body>
+    <section id="sectionTop" class="section-top">
+        <div class="icon-and-extra">
+            <div id="icon0">
+            </div>
+            <div class="extra-information-tablet">
+                <p id="humidity_a"></p>
+                <p id="windSpeed_a"></p>
+                <p id="airPressure_a"></p>
```
Comment (HIPPIEKICK) [inline code/index.html:17]: camelCase ;)

File: code/script.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+    //Update sunrise and sunset information
+    const sunriseTime = new Date(json.sys.sunrise * 1000).toLocaleTimeString([], {hour: "2-digit", minute:"2-digit"});
+    sunrise.innerHTML = sunriseTime;
+
+    const sunsetTime = new Date(json.sys.sunset * 1000).toLocaleTimeString([], {hour: "2-digit", minute: "2-digit"});
+    sunset.innerHTML = sunsetTime;
+};
+
+
```
Comment (HIPPIEKICK) [inline code/script.js:61]: Unnecessary lines 😇

File: code/script.js [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+const getMaxTemperature = (json, dayNumber) => {
+    const day0Date = new Date();
+    const day0MillSeconds = day0Date.getTime();
+    const day1Date = new Date(day0MillSeconds + 86400000 * (dayNumber +1)).toLocaleDateString("sv-SE");
+    
+    const filteredForecast = json.list.filter(item => item.dt_txt.includes(day1Date));
+    const maxTempArray = filteredForecast.map(day => day.main.temp_max);
+    return Math.max(...maxTempArray);
+};
```
Comment (HIPPIEKICK) [inline code/script.js:83]: Some comments here would be nice 🙌

File: code/script.js [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+const getWeekday = (dayNumber) => {
+    const day0Date = new Date();
+    const day0MillSeconds = day0Date.getTime();
+    const day1Date = new Date(day0MillSeconds + 86400000 * (dayNumber + 1)).toLocaleDateString("en-US", {weekday : "short"});
+    return day1Date;
+};
```
Comment (HIPPIEKICK) [inline code/script.js:90]: Here too

**Comment by HIPPIEKICK** [comment]
> My reflections on how this week's project turned out: I'm happy with the outcome and feel that I've learned a lot about functions in JS and how to write shorter code. I only touched on the red level with the icons but would have done more if I had had more time. It didn't seem too difficult. In the blue level, I chose to show the min/max temperature for every day instead of choosing one specific time of day as suggested in the hint. That took me a while and felt like I created my own red level there :)
> 

That's the best kind of stretch goals - the ones you create for yourself! ⭐

**Pull request closed without merging**